src/annotator/frame-observer.js

```diff
@@ -1,16 +1,19 @@
 import debounce from 'lodash.debounce';
 
-import * as FrameUtil from './util/frame-util';
+export const DEBOUNCE_WAIT = 40;
 
 /** @typedef {(frame: HTMLIFrameElement) => void} FrameCallback */
 
-// Find difference of two arrays
-let difference = (arrayA, arrayB) => {
-  return arrayA.filter(x => !arrayB.includes(x));
-};
-
-export const DEBOUNCE_WAIT = 40;
-
+/**
+ * FrameObserver detects iframes added and deleted from the document.
+ *
+ * To enable annotation, an iframe must be opted-in by adding the
+ * `enable-annotation` attribute.
+ *
+ * Eventually we may want annotation to be enabled by default for iframes that
+ * pass certain tests. However we need to resolve a number of issues before we
+ * can do that. See https://github.com/hypothesis/client/issues/530
```

Comment:
Do you want me to remove the comment?

Reply:
Its probably worth keeping a record of why we don't automatically inject into iframes. Maybe change the comment to something like "We require the `enable-annotation` attribute to avoid the overhead of loading the client into frames which are not useful to annotate. See https://github.com/hypothesis/client/issues/530".